pkg/cases/handler.go

```diff
@@ -90,9 +118,9 @@ func (h *CaseHandler) NewPrescription(c echo.Context) error {
 // @Description
 // @Tags Case
 // @Produce json
-// @Param id path uint true "prescription ID"
+// @Param prescriptionID path uint true "prescription ID"
```

Comment:
Use one name. Either `prescriptionID` or `prescriptionid`, not both.
And don't forget the caseid